src/Searchbar.vue

```diff
@@ -49,6 +49,32 @@ export default {
       return value.replace(new RegExp(`(${phrase})`, 'gi'), '<mark>$1</mark>');
     },
   },
+  methods: {
+    down() {
+      if (this.current < this.items.length - 1) {
+        this.current += 1;
+        this.scrollWindow();
+      }
+    },
+    up() {
+      if (this.current > 0) {
+        this.current -= 1;
+        this.scrollWindow();
+      }
+    },
+    scrollWindow() {
+      const { dropdown } = this.$els;
+      const curEntry = dropdown.children[this.current];
+      const upperBound = dropdown.scrollTop;
+      const lowerBound = upperBound + dropdown.clientHeight;
+      const curBottom = curEntry.offsetTop + curEntry.offsetHeight;
```

Comment:
I think you should just name it literally to what it should have been provided by them: `currentEntryOffsetBottom`. Makes it more consistent.